CMakeLists.txt

```diff
@@ -9,6 +9,28 @@ project(bergamot_translator CXX C)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Note that with CMake MSVC build, the option CMAKE_BUILD_TYPE is automatically derived from the key
+# 'configurationType' in CMakeSettings.json configurations
+if(NOT CMAKE_BUILD_TYPE)
+  message(WARNING "CMAKE_BUILD_TYPE not set; setting to Release")
+  set(CMAKE_BUILD_TYPE "Release")
+endif()
+#MSVC can't seem to pick up correct flags otherwise:
+if(MSVC)
+  add_definitions(-DUSE_SSE2=1) # Supposed to fix something in the sse_mathfun.h but not sure it does
```

Comment:
@XapaJIaMnu I believe, you don't need to set this definition here because marian submodule already sets this if the compiler is MSVC ([see this](https://github.com/browsermt/marian-dev/blob/master/CMakeLists.txt#L140))

Reply:
Empirically, I think it does it if the generator in Ninja. In practise, this crashed every time for me when using the microsoft visual studio generator ;/. I think this would be fixed once this is fixed https://gitlab.kitware.com/cmake/cmake/-/issues/22133

Reply:
Okay.

EDIT: It seems to me that `marian-dev` submodule in this PR points to some other than `master` (please correct me if I am wrong). Is it intended?

Reply:
Hmmm, no... This is probably my bad, I will fix it before merging.